### DISCUSSION TITLE
Is there a way to get eslint to ignore long markdown lines?

### DISCUSSION THREAD

**Discussion by omidantilong** [mdx-js/mdx#2036] [Q&A] [answered]
Hey! Thanks for this plugin. It's working great so far :) 

Just a question around the `max-len` rule in eslint. It has a bunch of options but none of these is applied to long lines in mdx files. Long lines always fall foul of the rule - so my workaround for now to keep eslint happy has just been to add an override for .mdx files. But ideally I'd like to keep the rule in place, just not for markdown blocks. 

I hope that made sense. I've attached a screenshot which hopefully illustrates it a bit better. Cheers!

<img width="822" alt="Screenshot 2022-05-13 at 21 14 18" src="https://user-images.githubusercontent.com/1922098/168383337-a12b74d8-92f6-460c-ab38-a9c158638dab.png">

**Comment by JounQin** [comment]
@omidantilong A reproduction is needed.

**Comment by omidantilong** [comment, marked as answer]
@JounQin it actually looks like prettier does a good job handling line length in place of eslint here, with the rule turned off in eslint overrides. I'm happy with that for now. If you want a reproduction for your own investigations still let me know.

**Reply by JounQin** [reply]
It seems not a bug but a feature request, and as you said, `eslint-plugin-prettier` is much more suitable to be used in this case, so I'd like to not change anything.
